bitnami/wordpress/Chart.yaml

```diff
@@ -1,6 +1,6 @@
 apiVersion: v1
 name: wordpress
-version: 9.9.3
+version: 9.9.4
```

Comment:
This should be a minor bump

Reply:
Ok, so I assume that would mean 9.10.0 right ? changed